examples/build.gradle

```diff
@@ -33,6 +33,10 @@ dependencies {
     // examples/advanced need this for JsonFormat
     implementation "com.google.protobuf:protobuf-java-util:${protobufVersion}"
 
+    // examples/authentication need this to create and verify JSON Web Tokens (JWTs)
+    implementation "io.jsonwebtoken:jjwt:0.9.1"
+    implementation "javax.xml.bind:jaxb-api:2.3.1"
```

Comment:
As this introduces extra dependencies, would you move your example out of the project of `hellowrold` example to a standalone directory, e.g. **examples/example-jwt-authentication/**, following the same pattern as examples in https://github.com/grpc/grpc-java/tree/master/examples#other-examples ?

We want to keep our project of `helloworld` example with minimum dependencies.

Reply:
Done.